### PR TITLE
Round numbers for Incito

### PR DESCRIPTION
**Pull request by mortenbo**

### CODE REVIEW

**Review by klarstrup — CHANGES_REQUESTED** [review]
Me no comprendo, decimal pixel values are valid 🤔

**Comment by klarstrup** [comment]
If this somehow addresses a11y support issues we were talking about: Can we just do this on the incito generation side for problematic incitos instead? Then they retain control and the fix is out instantly rather than needing to propagate through the SDK

**Comment by mortenbo** [comment]
@klarstrup OK, reverted it now. They'll fix on Incito. But we would still like to clean the style.

File: lib/incito-browser/incito.ts [hunk 1]
```diff
@@ -283,7 +283,9 @@ function renderView(view, canLazyload) {
                     .forEach((style) => {
                         const {0: key, 1: value} = style.trim().split(':');
 
-                        styles[key] = value;
+                        if (key && key.length && value && value.length) {
```
Comment (klarstrup) [inline lib/incito-browser/incito.ts:294]: Don't need this length check, empty strings are falsey by themselves

File: lib/incito-browser/incito.ts [hunk 2]
```diff
@@ -283,7 +283,9 @@ function renderView(view, canLazyload) {
...
+                            styles[key] = value.trim();
```
Comment (klarstrup) [inline lib/incito-browser/incito.ts]: `value` could be an empty string after the `trim()`, consider trimming before the above `if `clause

**Review by klarstrup — APPROVED** [review]
Nits

File: lib/incito-browser/incito.ts (hunk 2 above)
Comment (mortenbo) [reply lib/incito-browser/incito.ts]: True - better now?

**Review by klarstrup — APPROVED** [review]

**Comment by mortenbo** [comment]
@martensievers Shall we still go with this in the SDK or will you fix it server-side?

**Pull request closed without merging**